src/ensemble/CavityEnsemble.h

```diff
@@ -59,7 +59,7 @@ class CavityEnsemble {
    int ownrank;  // for debugging purposes (indicate rank in console output)
    bool initialized;
    bool rotated;
-   unsigned interval;  // how often?
+   //unsigned interval;  // how often?
```

Comment:
See above. There is no need for the interval in CavityEnsemble to be any different from the writer, because if the intervals differ, the information is lost anyway.
I cant think of a use case for different intervals between the two.
Higher interval on the Ensemble -> Writer outputs the same data multiple times
Higher interval on the Writer -> Ensemble calculates cavities and then overwrites them again before ever being used/written.

Reply:
Changed as requested f33bb40b77a2d205356e573a1ebb480f39ef7b75